proto/capten_sdk.proto

```diff
@@ -7,6 +7,8 @@ package captensdkpb;
 service capten_sdk {
     rpc GetGitProjectById(GetGitProjectByIdRequest) returns (GetGitProjectByIdResponse) {}
     rpc GetContainerRegistryById(GetContainerRegistryByIdRequest) returns (GetContainerRegistryByIdResponse) {}
+    rpc AddPluginUsage(AddPluginUsageRequest) returns (AddPluginUsageResponse) {}
```

Comment:
change to "RegisterResourceUsage"
   string resourceType 
   string resourceId 
   string pluginName

Based on resourceType gitProject/conatinerregistry, it need update in corresponding registry

Same applied to "DeRegisterResourceUsage"